roles/network_plugin/calico/templates/calico-node.service.j2

```diff
@@ -20,6 +20,7 @@ ExecStart={{ bin_dir }}/calicoctl node run --ip={{ip | default(ansible_default_i
 ExecStart={{ bin_dir }}/calicoctl node run --ip={{ip | default(ansible_default_ipv4.address) }} --node-image={{ calico_node_image_repo }}:{{ calico_node_image_tag }} {% if calico_network_backend is defined %}--backend={{calico_network_backend }}{% endif %}
 {% endif %}
 {% endif %}
+
```

Comment:
Also I've found in the official docs that

> Since Ansible version 0.9, templates are loaded with trim_blocks=True.

It means that **the first newline after a block is removed**. So the issue was introduced by the new conditional on L20: https://github.com/kubernetes-incubator/kargo/pull/621/commits/f6d69d0a00dfbac4cb6a8a323050a27929a43d4b#diff-e63719c83990d084186fd0cbe9bc7ce2R20